math.md

```diff
@@ -2014,17 +1907,12 @@ func TestSVGWriterHourHand(t *testing.T) {
 ### Try to run the test
 
 ```
---- FAIL: TestSVGWriterHourHand (0.00s)
-    --- FAIL: TestSVGWriterHourHand/06:00:00 (0.00s)
-        clockface_acceptance_test.go:113: Expected to find the hour hand line {X1:150 Y1:150 X2:150 Y2:200}, in the SVG lines [{X1:150 Y1:150 X2:150 Y2:60} {X1:150 Y1:150 X2:150 Y2:70}]
-FAIL
-exit status 1
-FAIL	github.com/gypsydave5/learn-go-with-tests/math/v10/clockface	0.013s
+clockface_test.go:113: Expected to find the minute hand line {X1:150 Y1:150 X2:150 Y2:200}, in the SVG lines [{X1:150 Y1:150 X2:150 Y2:60} {X1:150 Y1:150 X2:150 Y2:70}]
 ```
 
 ### Write enough code to make it pass
 
-And we can now make our final adjustments to `svgWriter.go`
+And we can now make our final adjustments to the SVG writing constants and functions:
```

Comment:
The tutorial never mentioned a `svgWriter.go` file before. I'd suggest it mentioning it earlier, if the goal is for us to split the functions like that.